src/components/MealDetail.vue

```diff
@@ -0,0 +1,88 @@
+<template>
```

Comment:
Theoreically, I could use this component to display existing meals or create new ones. Right now it only does the latter.